napari/_qt/qt_main_window.py

```diff
@@ -157,7 +158,26 @@ def _add_file_menu(self):
         )
         open_folder.triggered.connect(self.qt_viewer._open_folder)
 
-        screenshot = QAction('Screenshot', self._qt_window)
+        save_all_layers = QAction('Save all layer(s)...', self._qt_window)
+        # Save ctrl+S for saving the entire viewer state?
+        save_all_layers.setShortcut('Ctrl+Shift+S')
+        save_all_layers.setStatusTip('Save all layers')
+        save_all_layers.triggered.connect(
+            lambda: self.qt_viewer._save_layers(selected=False)
+        )
+
+        save_selected_layers = QAction(
+            'Save selected layer(s)...', self._qt_window
+        )
+        # Save ctrl+S for saving the entire viewer state?
+        save_selected_layers.setShortcut('Ctrl+Alt+Shift+S')
+        save_selected_layers.setStatusTip('Save selected layers')
+        save_selected_layers.triggered.connect(
+            lambda: self.qt_viewer._save_layers(selected=True)
+        )
+
+        # kinda want this shortcut for other things...
```

Comment:
Hilarious. What things? I'm not highly attached to this shortcut, but perhaps @sofroniewn is...

Reply:
You mean `'Ctrl+Alt+S'` should not be for saving screenshot but something else? That's fine - do we have a suggestion for what saving screenshot should be? I'm open

Reply:
I mostly just want to discuss it.  That's a pretty nice key combo, and I wonder how often someone needs to save a screenshot.  However, since everyone was happy with using Ctrl+S for saving layers, we're not running out of save key-combos.  Maybe someday, Ctrl+Alt+S can be for saving viewer state :)

Reply:
:-) let's leave for now then

Reply:
maybe remove the comment :-)

Reply:
will do! :)